graphql-dgs-codegen-shared-core/build.gradle

```diff
@@ -21,12 +21,13 @@ plugins {
 }
 
 dependencies {
-    api platform('com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:5.5.1')
-    implementation platform('com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:5.5.1')
-    compileOnly platform('com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:5.5.1')
+    api platform('com.netflix.graphql.dgs:graphql-dgs-platform-dependencies:9.1.3')
```

Comment:
nit: is there a reason why not to use `ext {}` block with variables for versions for reusing in all `build.gradle` files?

Reply:
Other than not wanting to change more than needed for this PR, I'm not aware of any reason as such.